src/lib/components/buy_product/BuyProduct.js

```diff
@@ -7,6 +7,9 @@ import Column from '../column/Column';
 import AddToCart from '..//add_to_cart/AddToCart';
 import ImageGallery from '..//image_gallery/ImageGallery';
 import MoneyHelper from '../../utils/MoneyHelper';
+import BackToProducts from '../back_to_products/BackToProducts';
```

Comment:
I am not fully convinced on back to products button. which is nothing but just a button.